go/client/cmd_encrypt.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol"
+	"github.com/keybase/go-framed-msgpack-rpc"
+)
+
+type CmdEncrypt struct {
+	libkb.Contextified
+	filter       UnixFilter
+	recipients   []string
+	trackOptions keybase1.TrackOptions
+}
+
+func NewCmdEncrypt(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "encrypt",
+		ArgumentHelp: "<usernames...>",
+		Usage:        "Encrypt messages or files for keybase users",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdEncrypt{
+				Contextified: libkb.NewContextified(g),
+			}, "encrypt", c)
+		},
+		Flags: []cli.Flag{
```

Comment:
Can you alphabetize these flags?  I've been trying to do that recently... Just did it for PGP encrypt